src/tests/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../components/App";
+
+describe("App", () => {
+  const { asFragment } = render(<App />);
+
+  it("renders correctly", () => {
+    expect(asFragment()).toMatchSnapshot();
+  });
+});
```

Comment:
You should render the component inside each individual test. Also, since this test hadnt been run there was no snapshot created for it
```
describe("App", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});
```